src/cpu/aarch64/acl_gemm_convolution.hpp

```diff
@@ -140,6 +152,26 @@ struct acl_gemm_convolution_fwd_t : public primitive_t {
         acl_conv_gemm_conf_t acp_;
 
     protected:
+        bool output_scales_mask_ok() const {
+            using namespace data_type;
+            const auto &mask = attr()->output_scales_.mask_;
+            return IMPLICATION(!utils::one_of(src_type, s8, u8),
+                           attr()->output_scales_.has_default_values())
+                    // ACL only supports uniform or per-channel scaling
+                    && (mask == 0 || mask == 1 << 1);
```

Comment:
Is this really the case?

In the code below I see:
``` cpp
    // Add quantization info to tensors
    const float *scales = attr.output_scales_.scales_;
    if (acp.is_int8) {
        acp.src_info.set_quantization_info(arm_compute::QuantizationInfo(1, 0));
        acp.bia_info.set_quantization_info(arm_compute::QuantizationInfo(1, 0));
        acp.wei_info.set_quantization_info(arm_compute::QuantizationInfo(1, 0));
        acp.dst_info.set_quantization_info(
                arm_compute::QuantizationInfo(1.0f / scales[0], 0)); /// !!!
    }
```

Here, at `!!!` only the first value of output scales is taken. Do this mean that ACL only support a per-tensor scaling factor?

Reply:
Compute Library does support both per_tensor and per_scale, but my integration does *not* at the moment (however,  it's planned for the near future).

I understand though that my imposed limitation is larger than the real limitation. Will fix.